internal/worker/server.go

```diff
@@ -249,8 +249,8 @@ func (s *Server) ManifestByIdJobStatus(id uuid.UUID, result *ManifestJobByIDResu
 		return nil, nil, err
 	}
 
-	if jobType != "manifest-by-id" {
-		return nil, nil, fmt.Errorf("expected \"koji-init\", found %q job instead", jobType)
+	if jobType != "manifest-id-only" {
```

Comment:
ah good catch, I think this needs 'manifest-id-only' everywhere no? Also on line 269.

Reply:
Yeah good point! I actually think this function isn't used at all and only the other one has any references